site/index.php

```diff
@@ -0,0 +1,43 @@
+<?php
+include('config/init.php');
+function add($param1, $param2){
+  return $param1 + $param2;
+}
+function subtract($param1, $param2){
+  return $param1 - $param2;
+}
+function multiply($param1, $param2){
+  return $param1 * $param2;
+}
+function divide($param1, $param2){
+  return $param1 / $param2;
+}
+
+echo "
+  <form action='' method='get'>
+    <input name='param1' /> <br>
+    <input type='radio' name='add' /> add <br>
+    <input type='radio' name='subtract' /> subtract <br>
+    <input type='radio' name='multiply' /> multiply <br>
+    <input type='radio' name='divide' /> divide <br>
+    <input name='param2' /> <br>
+    <input name='submitForm' type='submit' />
+    </form>
+    ";
+    if(isset($_REQUEST['add'])){
+      echo "<h1>The answer is: ".add($_REQUEST['param1'],
+      $_REQUEST['param2'])."</h1>";
+    }
+    elseif(isset($_REQUEST['subtract'])){
+      echo "<h1>The answer is: ".subtract($_REQUEST['param1'],
+      $_REQUEST['param2'])."</h1>";
+    }
+    elseif(isset($_REQUEST['multiply'])){
+      echo "<h1>The answer is: ".multiply($_REQUEST['param1'],
+      $_REQUEST['param2'])."</h1>";
+    }
+    else {
+      echo "<h1>The answer is: ".divide($_REQUEST['param1'],
```

Comment:
This all looks good. You could slightly shorten the code by writing it like this (note: I'm assuming you've changed the radio list to have all options use the name "operator" but have different values):
```
if(isset($_REQUEST['operator'])){
    if($_REQUEST['operator'] == 'add'){
        $answer = add($_REQUEST['param1'], $_REQUEST['param2']);
    }
    else if($_REQUEST['operator'] == 'subtract'){
        $answer = subtract($_REQUEST['param1'], $_REQUEST['param2']);
    }
}
//Do multiply and divide
echo "<h1>The answer is: $answer";
```

This way you're only echoing the html once rather than having an echo statement in every single if statement.